documentation/Environments.md

```diff
@@ -0,0 +1,63 @@
+# Authentication service environments
+
+The auth service can support multiple environments, allowing it to service multiple hosts.
+Alternate environments only affect account login and linking - all the environments share
+tokens and are invisible from the perspective of a service contacting the auth server to look up
+users or validate tokens.
```

Comment:
This is confusing about being invisible.

Reply:
How about

```
Alternate environments only affect the account login and linking flows and are otherwise not relevant to the auth system once a login or account link is complete. For example, a service contacting the auth service to validate a login token does not need to concern itself about which environment was used to create the token, and the auth service itself does not record any information about which environment was used to create a login token or link an account.
```

Reply:
Definitely more clear, thanks!

Reply:
Updated